german/schwimmen/fluegel_freischneiden/fluegel_freischneiden.md

```diff
@@ -0,0 +1 @@
+Dort hinten, an den Klippen aus scharfkantigen Felsen wird sich etwas finden lassen. Dieser Stein ist perfekt. Endlich sind die Flügel frei. Erst einmal strecken und jetzt ...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.